### PR TITLE
Apply Portuguese version of all website updated

### PR DESCRIPTION
**Pull request by nazarepiedady**
Since no more people appeared to help me translate to Portuguese (not Brazilian) the documentation, I decide to do it myself and dedicate a little more time to have the first version of Portuguese of the entire website.

This pull request include the translation for almost folders and files with exceptions of:

- announcements
- case-studies
- contribution-guide
- tutorials

I am using this pull request to request to be added as one of responsible of Portuguese version of the entire website and since the majority of the website is ready I believe that the next build of the website should include the Portuguese version.

@smarroufin

### CODE REVIEW

**Comment by nazarepiedady** [comment]
@smarroufin could you make it happen?

**Comment by nazarepiedady** [comment]
@smarroufin could you make it happen?

File: content/pt/design/index.md [hunk 1]
```diff
@@ -1,28 +1,27 @@
 ---
 template: blank
-title: 'Design Kit'
-description: 'Learn how to use Nuxt logos, colors and fonts and download the design kit.'
+title: 'Kit de Design'
+description: 'Aprenda como usar os logos do Nuxt, colres e fontes e descarregue o kit de design.'
```
Comment (ChristopheCVB) [inline content/pt/design/index.md:4]: `cores`?

File: content/pt/index.md [hunk 2]
```diff
@@ -1,125 +1,125 @@
...
 
 #secondary-button
-:app-button[Get started]{ to="/docs/get-started/installation" }
+:app-button[Começar]{ to="/docs/get-started/installation" }
 ::
 
 ::home-learn-master
 ---
-category: Learn
+category: Aprender
```
Comment (ChristopheCVB) [inline content/pt/index.md:24]: @smarroufin are those supposed to be tranlsated?

**Review by ChristopheCVB — COMMENTED** [review]
Terrific work, but took me 2h+ to review. Consider splitting up into multiple PRs in the future if you can ;)
I just found some typos and questions to @smarroufin

File: content/pt/deployments/azure-portal.md [hunk 3]
```diff
@@ -40,88 +40,88 @@ start()
...
 export default {
 ... config
 }
 ```
 
-After:
+Depois:
 
-```
+```js
```
[the remark this answers is not in the document]
Comment (ChristopheCVB) [reply content/pt/deployments/azure-portal.md:57]: It does, I've seen it used elswhere

File: content/pt/index.md (hunk 2 above)
Comment (nazarepiedady) [reply content/pt/index.md:24]: I was using French version to inform myself what is supposed to be translated.

File: content/pt/deployments/index.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
 template: PageList
-title: Deployments
-description: "Extend and automate your workflow by using deployments for your favorite tools."
-heroTitle: Deployments
-heroDescription: "Extend and automate your workflow by using deployments for your favorite tools."
+title: Desdobramento
```
[the remark this answers is not in the document]
Comment (nazarepiedady) [reply content/pt/deployments/index.md:3]: I will correct it

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -19,8 +19,8 @@
 /i18n/id-ID.js      @febrihidayan
 /content/es/**      @MrJmpl3
 /i18n/es-ES.js      @MrJmpl3
-/content/pt/**      @ChristopheCVB
-/i18n/pt-PT.js      @ChristopheCVB
+/content/pt/**      @ChristopheCVB @nazarepiedady
```
[the remark this answers is not in the document]
Comment (nazarepiedady) [reply .github/CODEOWNERS:22]: I want to be one of the responsible of Portuguese version of the documentation.

File: content/pt/examples/5.seo/1.html-head.md [hunk 6]
```diff
@@ -1,29 +1,29 @@
...
+Neste exemplo:
 
-`nuxt.config.js` uses the `head` property to show a `title`, `titleTemplate`, and `meta` including `description`. It also shows how to add an external google font using the link property and some JS using the script property. The lang and amp attributes are set with the `htmlAttrs` property.
+O ficheiro `nuxt.config.js` usa a propriedade `head` para mostrar um `title` (título), `titleTemplate` (modelo do título), e `meta` incluindo o `description` (descrição). Ele também mostra como adicionar uma fonte externa do Google usando a propriedade `link` e algum JavaScript usando a propriede `script`. Os atributos `lang` e `amp` são definidos com a propriedade `htmlAttrs`.
 
-`pages/index.vue` uses the head property as a function with dynamic data and uses the google font.
+O `pages/index.vue` usa a propriedade `head` como uma função com dados dinâmicos e usa o Google Font.
 
-`pages/about.vue` uses the head property as an object.
+O `pages/index.vue` usa a propriedade `head` como um objeto.
```
Comment (smarroufin) [inline content/pt/examples/5.seo/1.html-head.md:19]: `pages/index.vue` -> `pages/about.vue`

File: .github/CODEOWNERS (hunk 5 above)
[the remark this answers is not in the document]
Comment (smarroufin) [reply .github/CODEOWNERS:22]: Nice to hear you want to contribute actively and review. I note that.
This file is only edited by the repo maintainers and we will make this change in another PR. Please discard this change.

**Review by smarroufin — CHANGES_REQUESTED** [review]
This is quite a massive PR. I'll join @ChristopheCVB on that, please split it into smaller ones, by folders at least. This is insanely painful to review and could only make the approval/review process longer.

However, great work! I'll see if we can put PT live soon, I guess we are close to 100% translation on the PT folder.

Thanks for all the work 💚

**Comment by nazarepiedady** [comment]
@smarroufin and @ChristopheCVB thanks for your needed reviews, I am closing this pull request to start a small one for each part of the translated documentation.

**Pull request closed without merging**